pom.xml

```diff
@@ -99,7 +99,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <version>2.26-beta-1</version>
+            <version>2.26</version>
```

Comment:
This version is blacklisted on the update center right now, but the dependency is test-scope only and the Maven release is totally fine, so it shouldn't cause any issues.